src/redux/reducers/cryptoReducers.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { combineReducers } from 'redux';
+
+const loadCrypto = createAsyncThunk('fetchData',
+  async (current) => {
+    const result = await fetch(`https://api.coinlore.net/api/tickers/?start=0&limit=${current}`, { method: 'GET' });
+    const res = await result.json();
+    return res.data;
+  });
```

Comment:
- [ ] Kindly add mock test for the fetch function you have created as per the project requirement